database/database.go

```diff
@@ -35,7 +35,7 @@ func migrateDB(db *gorm.DB) {
 
 	err = goose.Up(sqlDB, "database/migrations", goose.WithAllowMissing())
 	if err != nil {
-		log.Fatalf("Failed to migrate database: %v", err)
+		log.Println(err)
```

Comment:
```suggestion
		log.Errorf("Failed to migrate database: %v", err)
```

If we use "Error", normally it will not shutdown the app. But at least it remains treated as an error message :)

Reply:
@malparty Fixed it here 🙏 
[34877e7](https://github.com/markgravity/golang-ic/commit/34877e76d14e3e95e4c905b3f76feb2e8c44ebbb)